### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by johangiraud**

### CODE REVIEW

File: static/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+.profile main {
+    display: block;
+}
+
+.profile h1 {
+    border: 0.3rem solid #00283C;
+    padding: 5rem;
+    text-align: center;
+    grid-area: title;
```
Comment (vinyll) [inline static/css/styles.css]: 💟

File: static/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.profile main section {
+    padding: 1.5rem;
+}
+
+.profile main section h2{
```
Comment (vinyll) [inline static/css/styles.css]: space before ` {`

File: static/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    text-align: center;
+}
+
+img.profile-pic {
```
Comment (vinyll) [inline static/css/styles.css]: Why do you need a class for this?
Can't you do `.profile img` instead?

File: static/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    display: block;
+    border-radius: 50%;
+    max-width: 20rem;
+    margin: 0.2rem auto;
+}
+.profile ul:first-of-type {
```
Comment (vinyll) [inline static/css/styles.css]: do you have multiple `ul` within `.profile`?

File: static/css/styles.css [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    width: 24rem;
+    margin: 5rem auto;
+}
+.profile [class^="icon-"],
+.profile [class*=" icon-"] {
```
Comment (vinyll) [inline static/css/styles.css]: I like this 🥇

File: static/css/styles.css [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+@media screen and (min-width: 750px) {
+    .profile main {
+        display: grid;
+        grid-gap: 2rem;
+        grid-template-columns: 50% 50%;
+        grid-template-areas:    'title title'
+                                'content sidebar';
+        margin-right: 2rem; /* don't understand why I need that. Otherwise h1 doesn't have a right margin */
```
Comment (vinyll) [inline static/css/styles.css]: Weird, no clue…

File: templates/base.html [hunk 7]
```diff
@@ -4,9 +4,12 @@
 <meta name=description content="{% block meta_description %}{% endblock %}">
 
 <link rel=stylesheet href=https://rawgit.com/cafebabel/styleguide/master/styleguide.css>
+<link rel=stylesheet href=/static/css/styles.css>
 <link rel=stylesheet href=/static/css/edit.css>
 <script src=/static/js/utils.js></script>
 
+<body class="{% block body_class%}{% endblock %}">
```
Comment (vinyll) [inline templates/base.html]: for consistency, insert a space `…_class %}`

File: templates/profile.html [hunk 8]
```diff
@@ -1,40 +1,44 @@
 {% extends 'base.html' %}
 
 {% block body %}
+{% block body_class %}profile{% endblock %}
```
Comment (vinyll) [inline templates/profile.html]: This should be outside of the `{% block body %}`. Probably above.

File: templates/profile.html [hunk 9]
```diff
@@ -1,40 +1,44 @@
...
-
-    <h2>Social accounts</h2>
-    <ul>
-      <li>Twitter: <span data-edit-name=socials-twitter>{{ user.profile.socials.twitter }}</span></li>
-      <li>Facebook: <span data-edit-name=socials-facebook>{{ user.profile.socials.facebook }}</span></li>
-      <li>Medium: <span data-edit-name=socials-medium>{{ user.profile.socials.medium }}</span></li>
-      <li>Flickr: <span data-edit-name=socials-flickr>{{ user.profile.socials.flickr }}</span></li>
-      <li>Pinterest: <span data-edit-name=socials-pinterest>{{ user.profile.socials.pinterest }}</span></li>
-      <li>Instagram<span data-edit-name=socials-instagram>{{ user.profile.socials.instagram }}</span></li>
+    <img src=https://rawgit.com/cafebabel/styleguide/master/img/profile-pic-placeholder.jpg alt="Profile pic placeholder" class=profile-pic> <!-- #US14 https://github.com/cafebabel/cafebabel.com/issues/14 -->
```
Comment (vinyll) [inline templates/profile.html]: Is this a temporary preview for future story?

File: static/css/styles.css (hunk 3 above)
Comment (johangiraud) [reply static/css/styles.css]: if we have contributions written (right column on desktop), we'll have cover images. And I don't want them to behave the same way as profile pic. Should I use `.profile img:first-of-type` instead ?

File: static/css/styles.css (hunk 4 above)
Comment (johangiraud) [reply static/css/styles.css]: I do.

File: static/css/styles.css (hunk 5 above)
Comment (johangiraud) [reply static/css/styles.css]: 👍

File: static/css/styles.css (hunk 6 above)
Comment (johangiraud) [reply static/css/styles.css]: maybe I can talk about this with Eric if I work with him a bit

File: templates/base.html (hunk 7 above)
Comment (johangiraud) [reply templates/base.html]: ok

File: templates/profile.html (hunk 9 above)
Comment (johangiraud) [reply templates/profile.html]: oui !

File: static/css/styles.css (hunk 2 above)
Comment (johangiraud) [reply static/css/styles.css]: rooo

File: static/css/styles.css (hunk 3 above)
Comment (vinyll) [reply static/css/styles.css]: It just feels redundant to have profile > img > profile-pic.
Eventually the solution you suggest may feel more pertinent.

File: templates/profile.html (hunk 8 above)
Comment (johangiraud) [reply templates/profile.html]: ok

File: static/css/styles.css (hunk 3 above)
Comment (johangiraud) [reply static/css/styles.css]: ok I do that

File: static/css/styles.css (hunk 4 above)
Comment (vinyll) [reply static/css/styles.css]: ohhh, `.profile` is the whole page, not just a component-style block, right…
What do you think of calling it `.profile-page` so we know its not reusable?
Just a suggestion…

Comment (johangiraud) [reply static/css/styles.css]: sure ! :-)

**Review by vinyll — APPROVED** [review]

File: static/css/styles.css [hunk 10]
```diff
@@ -43,57 +43,57 @@ summary {
 
 @font-face {
     font-family: 'vremena_grotesk';
-    src: url('./fonts/vremenagroteskbook-webfont.woff2') format('woff2'),
-    url('./fonts/vremenagroteskbook-webfont.woff') format('woff');
+    src: url('/static/fonts/vremenagroteskbook-webfont.woff2') format('woff2'),
```
Comment (vinyll) [inline static/css/styles.css]: Prefer relative urls: `../fonts/`

**Review by vinyll — CHANGES_REQUESTED** [review]

File: static/css/styles.css (hunk 10 above)
Comment (johangiraud) [reply static/css/styles.css]: yes, won't it then refer to localhost/5000/fonts/vremena.. or not ?

Comment (johangiraud) [reply static/css/styles.css]: ha non #sorry 😉  done.

File: static/css/styles.css [hunk 11]
```diff
@@ -611,6 +618,23 @@ button.menu-button + nav a {
     max-width: 20rem;
     margin: 0.2rem auto;
 }
+
+.profile-page main section:first-of-type h3 {
+    font-weight: 100;
+    font-size: 1.4rem;
+}
+
+.profile-page main section:nth-of-type(2) article{
```
Comment (vinyll) [inline static/css/styles.css:627]: space before ` {` ;)

**Review by vinyll — APPROVED** [review]

File: static/css/styles.css (hunk 11 above)
Comment (johangiraud) [reply static/css/styles.css:627]: yes, i made the correction it in the next PR

**Pull request merged**